specification/common-types/resource-management/v4/types.json

```diff
@@ -648,6 +648,35 @@
       "maxLength": 90,
       "x-ms-parameter-location": "method"
     },
+    "ManagementGroupNameParameter": {
+      "name": "managementGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the management group. The name is case insensitive.",
+      "minLength": 1,
+      "maxLength": 90,
+      "x-ms-parameter-location": "method"
+    },
+    "ScopeParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The scope at which the operation is performed.",
```

Comment:
I wonder if we have to declare anything about HTTP path escaping on a path-y parameter like this. Any SDK folks know?

Reply:
to support a path-y parameter, we must declare `x-ms-skip-url-encoding: true` https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-skip-url-encoding